src/components/LoaderRibbon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { makeStyles, Theme, createStyles } from "@material-ui/core/styles";
+import LinearProgress from "@material-ui/core/LinearProgress";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: "100%",
+      "& > * + *": {
```

Comment:
wow :))))

Reply:
Exactly :) It is however part of the official Material-UI [`linear-indeterminate`](https://material-ui.com/components/progress/#linear-indeterminate) widget implementation, so I left it as it is.